includes/discovery/os/trapeze.inc.php

```diff
@@ -0,0 +1,14 @@
+<?php
+//
+// Discovery of Juniper Wireless (Trapeze) devices.
+//
+if (!$os) {
+    if (preg_match('/^Juniper Networks/', $sysDescr)) {
+        if (str_contains($sysDescr, 'MX-')) {
+            $os = 'trapeze';
+        }
+        if (str_contains($sysDescr, 'WLC-')) {
```

Comment:
str_contains can accept arrays so you may as well put `array('MX-', 'WLC-')` into the first `str_contains` check.